bazel/README.md

```diff
@@ -26,6 +26,11 @@ The SHA1 hash in the above `strip_prefix` and `url` parameters correspond to the
 newer versions, you'll need to update those. To make use of older versions, change the
 parameter of `emsdk_emscripten_deps()`. Supported versions are listed in `revisions.bzl`
 
+Bazel 7+ additionally requires `platforms` dependencies in the `MODULE.bazel` file.
+```starlark
+bazel_dep(name = "platforms", version = "0.0.9")
+```
+
```

Comment:
```suggestion
```

This isn't needed: Bzlmod resolves transitive dependencies via Minimum Version Selection, so adding the `platforms` dep to the emsdk's `MODULE.bazel` file is sufficient for dependent modules to pick it up.

Reply:
I see, this snippet is meant for users pulling in `emsdk` via WORKSPACE but who do have `--enable_bzlmod` set, e.g. because it is the default in Bazel 7+? That makes sense if you don't want to fully set up `emsdk` for Bzlmod.

If you do, then this wouldn't be necessary and such users could just load `emsdk` via a `bazel_dep` in MODULE.bazel instead. But that could be done in a separate PR.